assets/sass/components/card.component.scss

```diff
@@ -1,25 +1,171 @@
 @use "../_func.scss" as *;
 @use "card.module.scss" as *;
 @use "../elements/dialog.scss" as *;
+@use "../elements/popover.scss" as *;
 @use "../elements/buttons.scss" as *;
 
 :host {
   --card-right-padding: 1rem;
+  //--card-head-height: 10rem;
+  --card-icon-right: 1rem;
 
-  --card-head-height: 10rem;
+}
+
+:host(:is(:hover,:focus,.hover):not(.prevent-hover)) {
+  --hover-outline-colour-default: var(--colour, var(--colour-primary));
+  outline: 2px solid var(--hover-outline-colour, var(--hover-outline-colour-default));
+  outline-offset: -2px;
+}
+
+:host(:is(:active,.active):not(.prevent-hover)) {
+  --card-icon-right: var(--card-icon-right-overide,0.5rem);
+  outline: none;
+}
+
+:host(.prevent-hover){
 
-  font-size: 1.5rem;
-  --line-height: 2rem;
+  pointer-events: none;
+
+  * {
+    pointer-events: all;
+  }
+}
+
+// #region Right arrow
+:host(.show-icon) .card__body {
+  --card-right-padding: 3.5rem;
+  
+  &:before,
+  &:after {
+    content: "";
+    position: absolute;
+    right: var(--card-icon-right);
+    bottom: var(--card-bottom-padding);
+    height: rem(24);
+    width: rem(24);
+    background: var(--colour-warning);
+    border-radius: 50%;
+  }
+
+  &:after {
+    background: var(--colour-primary-theme);
+    mask-image: var(--icon,var(--icon-arrow));
+    mask-size: 50%;
+    mask-repeat: no-repeat;
+    mask-position: 50% 50%;
+    -webkit-mask-image: var(--icon,var(--icon-arrow));
+    -webkit-mask-size: 50%;
+    -webkit-mask-repeat: no-repeat;
+    -webkit-mask-position: 50% 50%;
+    z-index: 2;
+  }
+}
+
+:host(:is(:hover,:focus,.hover):not(.prevent-hover):not([class*="colour-"])) {
+  
+  .card__body {
+
+    &:before {
+      background: var(--hover-icon-bg, var(--colour-primary-theme));
+    }
+    &:after {
+      background: var(--hover-icon-colour, #ffffff);
+    }
+  }
+}
+
+:host(:is(:active, .active):not(.prevent-hover):not([class*="colour-"])){
+
+  .card__body:before {
+    background: var(--hover-icon-bg, var(--colour-primary-theme));
+  }
+  .card__body:after {
+    background: var(--hover-icon-colour, #ffffff);
+  }
 }
+// #endregion
 
+// #region illustration
+.card__body {
+  &:has(.card__illustration) {
+
+    align-items: center;
+    display: flex;
+
+    .card__illustration {
+      height: rem(64);
+      width: rem(64);
+      margin-right: rem(24);
+      margin-top: rem(-20);
+      margin-bottom: rem(-20);
+      position: relative;
+
+      img {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        object-fit: contain;
+      }
+    }    
+  }
+}
+
+// #endregion
+
+// #region border left
+:host([class*="colour-"]) {
+  
+  border-left: rem(12) solid var(--colour,var(--colour-primary));
+}
+// #endregion
+
+// #region secondary button
+::slotted([slot="secondary"]) {
```

Comment:
are the !important neccessary, I can't remember recently but in the past i've had to override bits that use !important here

Reply:
Yeah unfortunately they are as the specificity doesn't seem to apply to slotted items. Think the best option is to avoid changing the style of slotted items. Sometimes this isn't possible and we will have to have the importants.

I this case though we could do something like this instead:

`slot[name="secondary"] {
  display: block;
  position: absolute;
  top: 0.5rem;
  right: 0;
  margin: 0.5rem 0.5rem 0 0;
  z-index: 999;
}`

and also add a part of secondary to the slot so that we can override it in the page.